Models/Login.cs

```diff
@@ -0,0 +1,21 @@
+﻿using MediatR;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MVCCore
+{
+    public class Login : IRequest<bool>
+    {
+        [Key]
+        public string userId { get; set; }
```

Comment:
Property of type string as the primary key will not degrade the performance, can we have another Id as identity